.github/workflows/ci.yml

```diff
@@ -56,6 +56,8 @@ jobs:
         sudo rm /etc/apt/sources.list.d/* && sudo dpkg --clear-avail # Speed up installation and get rid of unwanted lists
         sudo apt-get update
         sudo apt-get install -y --no-install-recommends libxmu-dev libxi-dev libgl-dev libglu1-mesa-dev libgles2-mesa-dev libwayland-dev libxkbcommon-dev libegl1-mesa-dev libosmesa6 mesa-utils libglvnd-dev
+        # Disable VM sound card because it is unreliable.
+        sudo sh -c 'echo "pcm.!default { type plug slave.pcm \"null\" }"  >> /etc/asound.conf'
```

Comment:
Can you put this in a separate step?

Reply:
I just did, but I need to merge this to another PR to check if it works, since this PR won't run the Ubuntu checks.

(Perhaps changes to the .github/workflows should always run the whole workflow?)

Reply:
I mean, if you tested the original PR I don't see the need

Reply:
I want to make sure it still works. I'm testing it in #7947 right now.

Reply:
I watched the logs run by, and the sound card change worked. All tests passed. I am satisfied.

I'll leave these changes merged into #7947 so I can watch them from time to time.